R/pandoc.R

```diff
@@ -46,6 +46,10 @@ pandoc_convert <- function(input,
                            verbose = FALSE,
                            wd = NULL) {
 
+  # ensure pandoc is available
+  if (!pandoc_available()) 
+    stop2("pandoc not found, ensure pandoc is installed")
+  
   # ensure we've scanned for pandoc
   find_pandoc()
```

Comment:
Thanks ! 

`pandoc_available()` is calling `find_pandoc()` so we can probably remove the call after. 

Also `pandoc_available(error = TRUE)` will throw an error. So I believe this could probably be replaced by 
```suggestion
  # ensure pandoc is available
  pandoc_available(error = TRUE)
```